### DISCUSSION TITLE
What's the procedure for automatically including members in given teams?

### DISCUSSION THREAD

**Discussion by ferrlen** [kimai/kimai#3136] [Questions & Answers] [answered]
Problem: we have multiple separate teams and a newly registered user isn't automatically assigned to any of them.

Question: When a new user registers, what would be the best approach to let them choose their teams? Right now, we are manually adding them to each team. 

Possible solution?: I thought of creating a HTML form for users and then submitting the input to a program running on a server that will register the new users using the Rest API.

Thanks!

**Comment by kevinpapst** [comment, marked as answer]
We need a new config setting that allows to configure a default team for new users.

The other option is to use the API.
